examples/slash_commands/main.go

```diff
@@ -12,28 +12,24 @@ import (
 	"github.com/bwmarrin/discordgo"
 )
 
-// Bot parameters
+// Parse the bot's parameters from the command line.
+// NOTE: Don't commit the BOT TOKEN on Git. This will give FULL ACCESS to your bot (to whoever can access your project).
+// Use an environment variable instead.
 var (
 	GuildID        = flag.String("guild", "", "Test guild ID. If not passed - bot registers commands globally")
 	BotToken       = flag.String("token", "", "Bot access token")
 	RemoveCommands = flag.Bool("rmcmd", true, "Remove all commands after shutdowning or not")
 )
 
-var s *discordgo.Session
-
-func init() { flag.Parse() }
-
-func init() {
-	var err error
-	s, err = discordgo.New("Bot " + *BotToken)
-	if err != nil {
-		log.Fatalf("Invalid bot parameters: %v", err)
-	}
-}
-
 var (
 	integerOptionMinValue = 1.0
 
+	// The discord session that represents your bot.
+	s *discordgo.Session
+
+	// Create the application commands (slash commands) the bot will add.
```

Comment:
If you are referring to 27, the comment connects the concept of a session with the bot. This example uses `s` to refer to the session, but `s` could be replaced with `bot` and the code will still make sense. Otherwise, line 30 has been changed to the pattern you suggested ("The application...") which describes the commands variable.